pom.xml

```diff
@@ -145,6 +145,18 @@
         <groupId>org.slf4j</groupId>
         <artifactId>log4j-over-slf4j</artifactId>
         <version>1.6.6</version>
+	<exclusions>
+          <!-- slf4j-api is provided by storm-core. Including slf4j-api
+	    here would cause its classes to be shaded preventing itself
```

Comment:
I feel like the description in this comment slightly conflicts with the change in `storm/pom.xml` below (which removes the shading of `org.slf4j`).  i.e., do we need both of these changes?